cobc/scanner.l

```diff
@@ -1271,11 +1271,71 @@ error_literal (const char *type, const char *literal)
 	cb_error ("%s", err_msg);
 }
 
+#define WITH_EBCDIC_LITERALS 1
```

Comment:
Why use both a macro and a dynamic flag ?

Reply:
I initially did not know to what extent this change would break scanning of literals from other standards. In particular, as of now any alphanumeric literal that is immediately followed by a digit without a separator triggers a warning or error if the `gcos-ebcdic-literals` feature is not enabled. In such a situation I though it's better to be able to preproces it out, so I left the macros.
However, using another variant of `CB_CONFIG_*` I'm still not aware of may be more suitable to silently disable ebcdic-scanning; I'll look it up.

Reply:
Ok now `gcos-ebcdic-literals` is boolean so the behavior is as before except for `gcos` dialects.

Reply:
Yes that's already done (that change is outdated).